### PR TITLE
Updating error message and indentation

### PR DESCRIPTION
**Pull request by coreydaley**
Updating error message to exclude usage of protoc newer than the 3.0* range.

Also updated some indentation

### CODE REVIEW

File: hack/update-generated-protobuf.sh [hunk 1]
```diff
@@ -26,11 +26,11 @@ os::build::setup_env
 genprotobuf="$( os::build::find-binary genprotobuf )"
 
 if [[ -z "${genprotobuf}" ]]; then
-	echo "It looks as if you don't have a compiled genprotobuf binary."
```
Comment (stevekuznetsov) [inline hack/update-generated-protobuf.sh]: Leave the unrelated changes out of this PR.

File: hack/update-generated-protobuf.sh [hunk 2]
```diff
@@ -2,8 +2,8 @@
 source "$(dirname "${BASH_SOURCE}")/lib/init.sh"
 
 if [[ -z "$(which protoc)" || "$(protoc --version)" != "libprotoc 3.0."* ]]; then
-  echo "Generating protobuf requires protoc 3.0.0-beta1 or newer. Please download and"
-  echo "install the platform appropriate Protobuf package for your OS: "
+  echo "Generating protobuf requires protoc >= 3.0.0-beta1, protoc < 3.1. Please"
```
Comment (stevekuznetsov) [inline hack/update-generated-protobuf.sh]: Sync this with the actual test above:
```
... requires protoc 3.0.? ....
```

Comment (coreydaley) [inline hack/update-generated-protobuf.sh]: Can you elaborate? Are you just wanting me to simplify the wording to something like 3.0.x ? (Assuming that nobody will try to use the 3.0.0-alpha(x) versions)

**Comment by coreydaley** [comment]
@stevekuznetsov Corrected as per your suggestions.

**Comment by stevekuznetsov** [comment]
We can grab the formatting changes in a larger overhaul later. [merge]

**Comment by marun** [comment]
networking job flake #11452

**Comment by coreydaley** [comment]
re[test]

**Comment by coreydaley** [comment]
[test]

**Comment by coreydaley** [comment]
[test]

**Comment by coreydaley** [comment]
[test]

**Comment by danmcp** [comment]
[merge]
